nautobot/extras/api/views.py

```diff
@@ -784,12 +792,6 @@ def run(self, request, *args, pk, **kwargs):
         if not get_worker_count(queue=task_queue):
             raise CeleryWorkerNotRunningException(queue=task_queue)
 
-        if job_model.supports_dryrun and job_model.read_only and cleaned_data.get("dryrun", False) is not True:
-            err_msg = (
-                "Unable to run or schedule job: This job is marked as read only and may only run with dryrun enabled."
-            )
-            raise ValidationError({"data": {"dryrun": [err_msg]}})
-
```

Comment:
This was a change from a behavior that was implemented in the feature branch. Since it never made it to `next` I didn't document its removal. We decided in a design meeting that `read_only` should be an attribute that the job author can set but it can't be overridden in the database and doesn't change the behavior of core. It's now just a flag to say "this job doesn't make any changes"

Reply:
Ack, thanks for the context.